### PR TITLE
Allow a buffer of time/retries between the DNS based maintenance page being switched off and the smoke tests testing the site

### PR DESCRIPTION
**Pull request by jaikoo**
Tweaking the retries for both Token request and Fetch email token

### CODE REVIEW

**Comment by jaikoo** [comment]
@threedaymonk can you take a look and sanity check?

File: smoke_test [hunk 1]
```diff
@@ -124,9 +125,13 @@ private
   end
 
   def request_token
-    visit '/'
```
Comment (threedaymonk) [inline smoke_test]: Does Capybara automatically visit the home page by default?

Comment (threedaymonk) [inline smoke_test]: To clarify a bit, it seems to me that `visit '/'` should be at the top, before the page content check, and the `visit '/'` at line 130 is redundant.

**Comment by threedaymonk** [comment]
Can you edit the title and description of this PR to better express the motivation for the change?

**Comment by jaikoo** [comment]
@threedaymonk sure, how does it look now?

**Comment by threedaymonk** [comment]
Better, but still needs the why? (If I recall correctly, that was because there's a variable delay between deployment succeeding and the site being accessible.)

**Comment by jaikoo** [comment]
@threedaymonk better?

**Pull request merged**